lib/hammer_cli_foreman/ssh.rb

```diff
@@ -12,6 +12,7 @@ class Command < HammerCLIForeman::Command
 
     command_name 'SSH to hosts'
     option %w(-c --command), 'COMMAND', _('Command to execute'), :attribute_name => :command, :required => true
+    option %w(-n --concurrent), 'CONCURRENCY', _('Number of concurrent SSH sessions'), :attribute_name => :concurrent
```

Comment:
I'd actually prefer if we didn't. It'd make the code more complex. When the option isn't set and no default is specified concurrent == nil. When concurrent_connections is set to nil it disables the concurrency limit. If we set a default value for concurrent it will be set to whatever the default is even if the option isn't specified. We'd either need to add an option to disable concurrency limits or use some other method to determine when the option wasn't actually passed by the user.